tests/main/hook-permissions/task.yaml

```
@@ -0,0 +1,44 @@
+summary: Ensure that post-refresh hook has permissions for its connected slot.
+
+# Note: it is critical for this test to use a slot that requires
+# peer=<plug security tag> expression label for dbus, and not simply
+# label=unconfined on a permanent slot snippet.
+details: |
+    Ensure that post-refresh hook has permissions to query upower-observe slot
+    and enumerate upower devices.
+
+systems: [-fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*]
```

would this be simpler just as a whitelist:
```suggestion
systems: [ubuntu-*, debian-*]
```

Also could we have a comment here about why it doesn't work on the the systems you have listed here now?

---

Actually this is because I originally intended to run this test only on systems where we have strict apparmor confinement (that should exclude debian, but I forgot it) - this is where the test makes the most sense. Anyway, I think there is no harm to run this test everywhere, so I've removed this and updated it to install upower where it's missing by default. Thanks for questioning this.